### PR TITLE
Add NairaEx

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -502,6 +502,14 @@ websites:
     regions:
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - email
```
Comment (phallobst) [inline _data/cryptocurrencies.yml]:
```suggestion
```
Does this refer to the "IP authorize via email" entry on the doc page? If so, it probably doesn't apply here.

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by 40504** [comment]
how PIN should be categorized?

**Comment by phallobst** [comment]
PIN is probably "something you know", i.e. similar to a password. That is not a second factor.

**Review by phallobst — APPROVED** [review]

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -502,6 +502,13 @@ websites:
     regions:
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - totp
+    doc: https://support.nairaex.com/support/solutions/articles/43000335554
+
```
Comment (ApCoder123) [inline _data/cryptocurrencies.yml:513]:
```suggestion
    regions:
      - ng

```

The website mentions 

> Nigeria's Premier Bitcoin Exchange

Is this site available outside of Nigeria?

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml [hunk 3]
```diff
@@ -502,6 +502,14 @@ websites:
...
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - totp
+    doc: https://support.nairaex.com/support/solutions/articles/43000335554
+    regions:
+      - ng
```
Comment (ApCoder123) [inline _data/cryptocurrencies.yml:512]:
```suggestion
      - ng

```
There should be a new line between entries.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**